Sources/CMakeLists.txt

```diff
@@ -8,6 +8,47 @@ See http://swift.org/LICENSE.txt for license information
 See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 ]]
 
+include(CMakeParseArguments)
+function(SwiftBuild_Bundle)
+  set(Options)
+  set(OneValueArguments MODULE)
+  set(MultiValueArguments FILES)
+  cmake_parse_arguments(PARSE_ARGV 0 BundleXCSpecs
+    "${Options}" "${OneValueArguments}" "${MultiValueArguments}")
+
+  list(TRANSFORM BundleXCSpecs_FILES PREPEND "${CMAKE_CURRENT_SOURCE_DIR}/")
+  add_custom_command(TARGET ${BundleXCSpecs_MODULE} POST_BUILD
+    COMMAND
+      ${CMAKE_COMMAND} -E make_directory "${CMAKE_CURRENT_BINARY_DIR}/SwiftBuild_${BundleXCSpecs_MODULE}.resources"
+    COMMAND
+      ${CMAKE_COMMAND} -E copy_if_different ${BundleXCSpecs_FILES} "${CMAKE_CURRENT_BINARY_DIR}/SwiftBuild_${BundleXCSpecs_MODULE}.resources/")
+
+  file(CONFIGURE
+    OUTPUT resource_bundle_accessor.swift
```

Comment:
I think that the absolute path with the conversion is important. Otherwise, the output location is not very well defined.

Reply:
Yes, it is documented to be relative to that, and I originally tried to use the relative path but ended up running into some issues. Furthermore, now that it is extracted into a helper, you no longer have a good idea of where `CMAKE_CURRENT_BINARY_DIR` is pointing to.

Reply:
ok, that makes sense to me, thanks. Updated the patch to use an absolute path here and avoid use of CMAKE_CURRENT_BINARY_DIR in general